app/dashboard/models.py

```diff
@@ -2236,9 +2237,26 @@ def view_props(self):
 
         return activity
 
+    @property
+    def either_view_props(self):
+        vp = self.cached_view_props
+        if not vp.get('pk'):
+            vp = self.view_props
+        return vp
+
+    def generate_view_props_cache(self):
+        self.cached_view_props = self.view_props
+        self.cached_view_props = json.loads(json.dumps(self.cached_view_props, cls=EncodeAnything))
```

Comment:
So sketchy... I should have never put this in the codebase in the first place LOL 😆

Reply:
it seems entirely ridiculous to make that the default json encoder doesnt encode datetimes..